lib/isaac_state.go

```diff
@@ -0,0 +1,24 @@
+// When a node receive ballot, if the ballotState is former then nodeRunnerState,
```

Comment:
When a node receive*s* a ballot, if the ballotState is *before* (or in this case, *lower*?) than nodeRunnerState, the ballot is ignored.